specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2020-06-25/guestconfiguration.json

```diff
@@ -965,7 +965,7 @@
       "properties": {
         "targetResourceId": {
           "type": "string",
-          "readOnly": true,
+          "readOnly": false,
```

Comment:
May I ask why this property is writeable? Per my understanding, service side already can get the vm from the parameter [VmNameParameter](https://github.com/Azure/azure-rest-api-specs/blob/4bab0bfdf7b1e2e3a50eeda420c2891d087d1d85/specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2020-06-25/guestconfiguration.json#L66), right?

Reply:
Ideally none of the properties are read-only from implementation perspective.
Not sure why historically when swagger was created, all these properties were marked as read-only.

This property user cannot pass in any way, so setting the flag to true.